invenio_theme/bundles.py

```diff
@@ -39,12 +39,12 @@
 from invenio_assets import BowerBundle
 
 css = BowerBundle(
-    'less/invenio_theme/styles.less',
-    filters='less, cleancss',
+    'scss/invenio_theme/styles.scss',
+    filters='sass, cleancss',
```

Comment:
This should be 'scss' instead of 'sass'. Syntax errors will otherwise pop up as this is using scss syntax rather than sass.